modules/project-docs/pages/sdk-release-notes.adoc

```diff
@@ -13,15 +13,26 @@ These pages cover the 4.x versions of the Couchbase PHP SDK.
 For download instructions, see the xref:sdk-full-installation.adoc[installation page].
 
 
-== Version 4.0.0 (XXX April 2022)
+== Version 4.0.0 (11 May 2022)
 
-Version 4.0.0 is the first major release of the next generation PHP SDK, built on the Couchbase++ library -- featuring multi-document distributed ACID transactions, and bringing a number of improvements related to internal connection behavior.
+Version 4.0.0 is the first major release of the next generation PHP SDK, built on the Couchbase++ library -- featuring
+multi-document distributed ACID transactions, and bringing a number of improvements related to internal connection
+behavior.
 
-API documentation: https://docs.couchbase.com/sdk-api/couchbase-php-client-4.0.0/
+API documentation: https://docs.couchbase.com/sdk-api/couchbase-php-client-4.0.0
 
-* https://issues.couchbase.com/browse/PCBC-123[PCBC-123]: 
-Sample ticket XXXXXX
+|===
+|Platform|
+|Source          |https://packages.couchbase.com/clients/php/couchbase-4.0.0.tgz[couchbase-4.0.0.tgz]
+|Checksum        |https://packages.couchbase.com/clients/php/couchbase-4.0.0.sha256sum[couchbase-4.0.0.sha256sum]
+|===
 
```

Comment:
@avsej, thanks for putting this up, do we not have a list of tickets (known issues, or fixed issues etc?)